docs/sources/k6/next/extensions/run/_index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: 'Run a test with extensions'
+menuTitle: 'Run a test with extensions'
+description: 'Run k6 with extensions to cover use cases not supported natively in the core.'
+weight: 03
+---
+
+# Run a test with extensions
+
+As listed in the [Explore extensions](https://grafana.com/docs/k6/<K6_VERSION>/extensions/explore) page, extensions fall into three categories. 
+
+How you run them depends on their category:
+
+- **Official & Community** → You can import them directly in your test. If your local binary doesn’t already include them, k6 automatically loads a compatible version when you run your test.
+- **Other** (from the ecosystem or your own) → These require building a custom k6 binary with [xk6](https://github.com/grafana/xk6).
+
+## Using automatic extension loading
```

Comment:
I didn't get into all the pragma things, etc. If someone has the time to add it, please, go ahead.